core/create.go

```diff
@@ -20,13 +23,24 @@ var (
 
 	// ErrThemeExists states that the specified theme already exists.
 	ErrThemeExists = errors.New("theme already exists, remove it first")
+
+	// ErrFileExist states that the specified file already exists.
+	ErrFileExists = errors.New("file already exists.")
```

Comment:
Errors do not have a dot.
```suggestion
	ErrFileExists = errors.New("file already exists")
```